src/veh_type.cpp

```diff
@@ -412,6 +505,18 @@ std::string vpart_info::name() const
     return name_;
 }
 
+int vpart_info::install_time( const Character &ch ) const {
+    ///\EFFECT_MECHANICS reduces time consumed installing vehicle parts
+    int lvl = std::min( ch.get_skill_level( skill_mechanics ).level(), MAX_SKILL );
+    return install_moves * ( 1.0 - ( lvl / 2.0 ) / MAX_SKILL );
+}
+
+int vpart_info::removal_time( const Character &ch ) const {
+    ///\EFFECT_MECHANICS reduces time consumed removing vehicle parts
+    int lvl = std::min( ch.get_skill_level( skill_mechanics ).level(), MAX_SKILL );
+    return removal_moves * ( 1.0 - ( lvl / 2.0 ) / MAX_SKILL );
```

Comment:
Duplicated function. I would be nice if `install_moves`/`removal_moves` could be passed as arguments to an extracted function.